src/darwin/Framework/CHIP/XPC Protocol/MTRXPCClientProtocol.h

```diff
@@ -27,6 +27,10 @@ MTR_NEWLY_AVAILABLE
 - (oneway void)deviceBecameActive:(NSNumber *)nodeID;
 - (oneway void)deviceCachePrimed:(NSNumber *)nodeID;
 - (oneway void)deviceConfigurationChanged:(NSNumber *)nodeID;
+
+@optional
+// temporarily optional to avoid lockstep needs
+- (oneway void)deviceInternalStateUpdated:(NSDictionary *)dictionary;
```

Comment:
```suggestion
- (oneway void)device:(NSNumber *)nodeID internalStateUpdated:(NSDictionary *)dictionary;
```

Reply:
Will change to this - something I realized later is that there's that actual individual ivar/property called `internalState`, so thought naming this `internalState` might be briefly confusing.  But with the types in the signature, I think it would draw people to look more closely and quickly understand.